clang-tools-extra/clang-tidy/tool/run-clang-tidy.py

```diff
@@ -462,6 +469,19 @@ def main():
         [make_absolute(entry["file"], entry["directory"]) for entry in database]
     )
 
+    # Filter source files from compilation database.
+    if args.source_filter:
+        try:
+            source_filter_re = re.compile(args.source_filter)
+        except:
+            print(
+                "Error: unable to compile regex from arg -source-filter:",
+                file=sys.stderr,
+            )
+            traceback.print_exc()
+            sys.exit(1)
```

Comment:
I were thinking more about something like:
```suggestion
        except Exception as e:
            print(
                "Error: unable to compile regex from arg -source-filter: %0" % (e),
                file=sys.stderr,
            )
            sys.exit(1)
```

Or something like that.

Reply:
I used `traceback.print_exc()` for the sake of consistency as it is already used in two other `except` blocks in this file.

Reply:
ok